app/move/controllers/create/save.js

```diff
@@ -17,6 +16,8 @@ function filterAnswer(currentAssessment, searchKey) {
 
 class SaveController extends CreateBaseController {
   async saveValues(req, res, next) {
+    const courtHearingService = req.services.courtHearing()
```

Comment:
I think it might be a cleaner transition to not require the services to be instantiated in each controller/middleware, rather we do that within the `set-services` middleware once the services support this way of being constructed.

What do you think?

Reply:
>I'm guessing this is being used as an example of how we can move to this approach?

I should've read the description properly 🤦